employee.rb

```diff
@@ -0,0 +1,13 @@
+class Employee 
+  attr_accessor :employee_id ,:employee_name, :age, :birth_date, :address, :department_id
+  @@no_of_employees=0
```

Comment:
this could have been an instance variable of company class
This is also fine though.